CMakeLists.txt

```diff
@@ -5,8 +5,8 @@ SET (boolinq_VERSION_MAJOR 2)
 SET (boolinq_VERSION_MINOR 0)
 
 
-SET (CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS} -O0 -ggdb3 -DDEBUG")
-SET (CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS} -O3")
+SET (CMAKE_CXX_FLAGS_DEBUG   "${CMAKE_CXX_FLAGS} -O0 -ggdb3 -DDEBUG -std=c++11")
+SET (CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS} -O3 -std=c++11")
```

Comment:
I can not pull this pull-request to proove it, but i gues the `-std=c++11` will not let me to build again. What about using `gnu++11`?